modules/alacritty/hm.nix

```diff
@@ -1,31 +1,60 @@
-{pkgs, config, lib, ... }:
-
-with config.stylix.fonts;
+# Documentation is available at https://alacritty.org/config-alacritty.html.
+{ config, lib, ... }:
 
 let
-  useYaml = (builtins.compareVersions config.programs.alacritty.package.version "0.13.0") < 0;
-  templateRepo = config.lib.stylix.templates.
-    "base16-alacritty${if useYaml then "-yaml" else ""}";
-
-  themeFile = config.lib.stylix.colors {
-    inherit templateRepo;
-  };
+  colors = config.lib.stylix.colors;
+  mnemonicsWithHashtag = builtins.mapAttrs (_: color: "#${color}") colors;
```

Comment:
Why not use `config.lib.stylix.colors.withHashtag` directly?

Reply:
`withHashtag` is pretty much defined exactly the same as `mnemonicsWithHashtag`, so yes, we should use that.

This was probably an oversight due to a lack of clear documentation on the subject.

Reply:
Yeah, it's an oversight on my part. While looking at the base16-nix source I missed that `withHashtag` iterates over based:

```nix
based = base-hex // base-hex-rgb // base-rgb-rgb
  // base-dec-rgb // base-hex-bgr // base-short
  // mnemonic;
withHashtag =
  let hashtag = color: "#${color}";
  in (lib.mapAttrs (_: hashtag) based) // { toList = map hashtag toList; };
```